tools/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+# SPDX-FileCopyrightText: 2023 Volker Krause <vkrause@kde.org>
+# SPDX-License-Identifier: MIT
+
+scriptdir=`dirname "${BASH_SOURCE[0]:-$0}"`
```

Comment:
`$BASH_SOURCE` isn't defined in an Alpine Linux `/bin/sh`, only in `/bin/bash`.

Reply:
Right, that's why there is the fallback to `$0`, which should be available everywhere (but might not contain the right thing if you get creative in calling the script).